internal/storage/index_storage.go

```diff
@@ -25,6 +25,11 @@ type IndexStorage interface {
 
 	GetPDVMeta(ctx context.Context, address string, id uint64) (*entities.PDVMeta, error)
 	SetPDVMeta(ctx context.Context, address string, id uint64, tx string, m *entities.PDVMeta) error
+
+	GetPDVDelta(ctx context.Context, address string) (float64, error)
+	GetPDVTotalDelta(ctx context.Context) (float64, error)
+	GetPDVRewardsDistributedDate(ctx context.Context) (time.Time, error)
+	SetPDVRewardsDistributedDate(ctx context.Context, date time.Time) error
```

Comment:
`SetPDVRewardsDistributedDate` is not used at the moment, but will be called as soon as PDV month rewards are distributed. It is going to be implemented in the next PR.